controller/models/apps/apps.go

```diff
@@ -101,11 +101,9 @@ func Logs(c *client.Client, appID string, lines int) (string, error) {
 		return "", fmt.Errorf(
 			`There are currently no log messages. Please check the following things:
 1) Logger and fluentd pods are running.
-2) If you just installed the logger components via the chart, please make sure you restarted the workflow pod.
-3) The application is writing logs to the logger component.
-You can verify that logs are appearing in the logger component by issuing the following command:
-curl http://<log service ip>:80/logs/%s on a kubernetes host.
-To get the service ip you can do the following: kubectl get svc deis-logger --namespace=deis`, appID)
+2) The application is writing logs to the logger component.
+3) Check that an entry in the ring buffer was created by checking the logger pod logs: kubectl logs <logger pod> --namespace=deis
+4) Making sure that the container logs were mounted properly into the fluentd pod: kubectl exec <fluentd pod> --namespace=deis ls /var/log/containers`, appID)
```

Comment:
is there a simple way to indicate what to look for in the above two sets of logs?